webpack.common.config.js

```diff
@@ -25,6 +25,7 @@ var entry = {
     // JS
     'base-page': staticPath('js/pages/base-page.js'),
     'home-page': staticPath('js/pages/home-page.js'),
+    'landing-page': staticPath('js/pages/landing-page.js'),
```

Comment:
> Nevermind, a decision was just made to remove /home.

@pattisdr Will that be a separate PR, then? This current PR is the one that *adds* `/home/` in the first place, right? Should we avoid adding it in the first place, then? :)